tests/fixtures/test_manager/invalid/schema/schemas/dns.yml

```diff
@@ -0,0 +1,10 @@
+---
+$schema: "http://json-schema.org/draft-07/schema#"
+$id: "schemas/dns_servers"
+description: "DNS Server Configuration schema."
+type: "object"
+properties:
+  dns_servers:
+    $ref: "../definitions/arrays/ip.yml#ipv4_hosts"
```

Comment:
If possible i think it would be best to not use external references for a fixture schema, we should be able to have everything define in a single file. It will help to reduce the number of files that we have to maintain

Reply:
I copied these files from `tests/fixtures/test_instances` to keep some separation. 

I initially allowed schema-enforcer to continue and skip the bad schema, so I wanted some good schemas to confirm. Since that's no longer the case, I'll reduce this down to only the bad schema.